testsuite/MDAnalysisTests/lib/test_nsgrid.py

```diff
@@ -371,3 +375,26 @@ def test_issue_2670():
 
     # should return the one atom overlap
     assert len(ag2.select_atoms('around 0.0 resid 3')) == 1
+
+
+reason = "Turned off by default. The test can be enabled by setting the ENABLE_HIGH_MEM_UNIT_TESTS " \
+         "environment variable. Make sure you have at least 10GB of RAM."
+
+
+# Tests that with a tiny cutoff to box ratio, the number of grids is capped
+# to avoid indexing overflow. Expected results copied from test_nsgrid_search
+# with no box.
+@pytest.mark.skipif(not os.getenv("ENABLE_HIGH_MEM_UNIT_TESTS"), reason=reason)
```

Comment:
This just checks if the env is set, if `ENABLE_HIGH_MEM_UNIT_TESTS=False` this will run as any non-empty string evaluates to True.

Reply:
Good point - I wasn't sure the best way to deal with this, what I ended up going to was distutils strtobool, but let me know if you have a better suggestion.